client/containers/Pub/PubDocument/PubDiscussions/Discussion/ThreadComment.tsx

```diff
@@ -84,7 +95,11 @@ const ThreadComment = (props: Props) => {
 						{isPreview ? ': ' : ''}
 					</span>
 
-					{isPreview && <span className="preview-text">{threadCommentData.text}</span>}
+					{isPreview && (
```

Comment:
⚠️ It's a cool idea to try to make sure the previews have RTL but sadly I don't think it's going to work without a way to truncate the contents of the editor a little bit more.

![image](https://user-images.githubusercontent.com/2208769/174105605-034a9c6f-1d76-4dc8-8eda-250b5dc6a8dc.png)

Otherwise we lose the "preview" nature of these boxes and they just contain the entire comment text, which could potentially be very long. This looks okay with one comment but will become unreasonable once there are many. There are some strategies we could try to limit their height:

1. Using `-webkit-line-clamp` to set a maximum number of lines to show in the element — [see our `clamp-lines` mixin](https://github.com/pubpub/pubpub/blob/cf76b61f7decc94fb78e68483cb2256c746dc4f5/client/components/UserNotifications/userNotifications.scss#L13). Didn't work for me when I tried it just now but it could be worth messing with a little more.
2. Some kind of reasonably short `max-height` on the editor, plus a nice transparent-to-opaque gradient over the remainder of the text so it fades gently into that good night. I feel like I see this all over the internet but I can't find a decent example right now, sorry.

But honestly, my vote is to revert to using `threadComment.text` and accept that we'll have left-aligned RTL text in the comment previews for now.

Reply:
This looks great! Except, the username and the preview text are now misaligned by 1px:

![image](https://user-images.githubusercontent.com/2208769/174127257-f04eb0de-b3a4-4595-972a-df3e7f285ff4.png)

but if you change `align-items` of the wrapping `.item-header` from `center` to `flex-start` it looks like that takes care of the problem:

![image](https://user-images.githubusercontent.com/2208769/174127580-d6192983-0413-48e6-9eca-f23a6ee9672d.png)